website/content/en/guides/developer/debugging/debugging.md

```diff
@@ -0,0 +1,760 @@
+---
+title: A guide for debugging Vector pipelines
+short: Debugging Vector
+description: Step by step guide for debugging pipelines
+tags: [ "dev", "debugging", "guides", "guide" ]
+author_github: https://github.com/pront
+domain: dev
+aliases: [ "/docs/guides/developer/config-autocompletion.md" ]
+weight: 1
+---
+
+# Debugging Guide
+
+This guide will describe an imaginary use case and how to progressively resolve issues using dev tools and Vector commands.
```

Comment:
```suggestion
This guide will describe an imaginary use case and how to progressively resolve issues using command line tools and Vector commands.
```

The usage of "dev" might scare off people that use Vector but aren't developers.